### ISSUE TITLE
aria-haspopup should have boolean value, has string like "dialog"

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#588]

**Comment by danielweck** [comment]
Apologies, ARIA 1.0 now supports enumerated values.
